.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+name: Release
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: "The type of release to make (Minor, Patch, etc)"
+        required: true
+        options:
+          - PATCH
+          - MINOR
+      release_title:
+        description: "The title of the release"
+        required: true
+
+jobs:
+  update-version:
+    runs-on: ubuntu-20.04 # latest
+    permissions:
+      contents: write # allow push
+
+    steps:
+    - name: Checkout Sources
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Make new release
+      run: |
+        ./publish-release.sh "${{ github.event.inputs.release_type }}" "${{ github.event.inputs.release_title }}"
```

Comment:
edge-case: is there an easy way to "escape" any weird characters in the release_title? I worry about a title like: `fix issues in "us-east-1"` turning into:
`./publish-release.sh "PATCH" "fix issues in "us-east-1""`
which is equivalent to
`./publish-release.sh "PATCH" "fix issues in " "us-east-1" ""`

if there's not an easy way ... don't go crazy trying to fix this edge case. This is an internal tool so we don't need to worry about attacks.  Maybe `publish-release.sh` could simply error out if more than 2 args are passed in? And if someone runs the workflow with a bad title then the workflow fails quickly, no biggie?

Reply:
Fixed! While it is just an internal tool, there *was* the chance someone could pass `$` as well, which would have been bad... It is not a problem anymore.